ompi/tools/mpirun/help-mpirun.txt

```diff
@@ -14,6 +14,10 @@
 Open MPI's mpirun command was unable to find an underlying prterun
 command to execute.  Consider setting the OMPI_PRTERUN environment
 variable to help mpirun find the correct underlying prterun.
+[prte-launch-failed]
+Open MPI's mpirun command was unable to launch the user's application 
+using the prterun command. This may indicate an issue with the prterun
+environment or incorrect configuration.
```

Comment:
Don't forget to add the error message line in here, and also remove the other 2 topics that are no longer relevant.